.github/workflows/ci.yml

```diff
@@ -106,8 +106,8 @@ jobs:
           restore-keys: ${{ runner.os }}-node-
       - run: npm install
       - env:
-          AXIOM_TOKEN: ${{ secrets.TESTING_CLOUD_DEV_ACCESS_TOKEN }}
-          AXIOM_URL: ${{ secrets.TESTING_CLOUD_DEV_DEPLOYMENT_URL }}
-          AXIOM_ORG_ID: ${{ secrets.TESTING_CLOUD_DEV_ORG_ID }}
+          AXIOM_TOKEN: ${{ secrets.TESTING_DEV_TOKEN }}
+          AXIOM_URL: ${{ secrets.TESTING_DEV_APP_URL }}
```

Comment:
Just be aware that if you do the cloud -> {app,api} split, this needs to become `API` instead of `APP`.

Reply:
where can I read more about the split? at first I used the API variable but tests failed.